dumpling-overview.md

```diff
@@ -233,7 +233,7 @@ export AWS_ACCESS_KEY_ID=${AccessKey}
 export AWS_SECRET_ACCESS_KEY=${SecretKey}
 ```
 
-Dumpling also supports reading credential files from `~/.aws/credentials`. For more information about URI parameter descriptions, see [URI Formats of External Storage Services](/external-storage-uri.md).
+Dumpling also supports reading credential files from `~/.aws/credentials` and using AWS profiles. For more information about URI parameter descriptions, see [URI Formats of External Storage Services](/external-storage-uri.md).
 
```

Comment:
for L239, do we need to provide two example commands so it is more clear for users? For instance:

- Use the default credentials from `~/.aws/credentials`:

```shell
tiup dumpling -u root -P 4000 -h 127.0.0.1 -r 200000 -o "s3://${Bucket}/${Folder}"
```

- Using a specific AWS profile:

```shell
export AWS_PROFILE=my-profile
tiup dumpling -u root -P 4000 -h 127.0.0.1 -r 200000 -o "s3://${Bucket}/${Folder}"
```

Reply:
thanks for suggestion. I will add it.

Reply:
I've added the examples.